src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/BaseFunction.php

```diff
@@ -31,7 +31,22 @@ abstract class BaseFunction extends FunctionNode
      */
     protected array $nodes = [];
 
-    abstract protected function customizeFunction(): void;
+    /**
+     * TODO Make abstract when {@see customiseFunction()} is no more.
+     */
+    protected function customizeFunction(): void
```

Comment:
```suggestion
    /* abstract */ protected function customizeFunction(): void
```

Reply:
CS fixer wants this comment on a separate line, which is why I didn't add it in the first place.

Reply:
Ah, yes, my old automation friend is sometimes overly prescriptive. It's good to ignore, then.